mno_analysis_tools/Pipfile

```diff
@@ -0,0 +1,22 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+rapidpro-python = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.6"
+
+[packages.coredatamodules]
```

Comment:
This is an unusual format - if `$ pipenv install` gave you this output can you let me know your `$ pipenv --version`?

Reply:
pipenv, version 2018.05.18

Reply:
Can you update to latest (2018.11.26)? Partly because that introduced a cleaner, more succinct syntax for git package declarations, and partly because early versions of pipenv are notoriously buggy.

Reply:
okay. thanks